client/pages/user-registration/registration.controller.js

```diff
@@ -25,14 +25,7 @@ angular.module('probedock.userRegistrationPage').controller('UserRegistrationPag
 
   $scope.$watch('organization.displayName', function(value) {
     value = value || '';
-
-    $scope.organizationNamePlaceholder = value
-      .replace(/[^a-z0-9\- ]+/gi, '')
-      .replace(/ +/g, '-')
-      .replace(/\-+/g, '-')
-      .replace(/\-+$/, '')
-      .replace(/^\-+/, '')
-      .toLowerCase();
+    $scope.organizationNamePlaceholder = api.slugify(value);
```

Comment:
This change to the registration form implementation is not sufficient. This is the code that sets the placeholder in the `name` field, computed from the `displayName` field, but what we actually want to do is remove the name field entirely, so this code should go as well.

Actually, [most of the code](https://github.com/probedock/probedock/blob/friendly-names/client/pages/user-registration/registration.controller.js#L26-L57) in the controller is obsolete if we only have the `displayName` field. [This part](https://github.com/probedock/probedock/blob/friendly-names/client/pages/user-registration/registration.controller.js#L13-L15) should be replaced with code similar to what you introduced in the model services.

The template should be updated to have only the `displayName` field, and to use the validation. Also, the organization part of the template currently has the display name and name on one line, and the public checkbox on another line. Since we're removing the name field, you can put the display name and public checkbox on the same line (with `.col-sm-6` divs).